src/View/Helper/FormHelper.php

```diff
@@ -15,25 +15,18 @@ class FormHelper extends Helper
     use OptionsAwareTrait;
 
     /**
-     * The default error style.
+     * The default feedback style.
      *
      * @var string
      */
-    public const ERROR_STYLE_DEFAULT = 'default';
+    public const FEEDBACK_STYLE_DEFAULT = 'default';
 
     /**
-     * The none error style.
+     * The tooltip feedback style.
      *
      * @var string
      */
-    public const ERROR_STYLE_NONE = 'none';
```

Comment:
I take it `none` isn't needed?

Reply:
Yeah, I [removed that style](https://github.com/FriendsOfCake/bootstrap-ui/pull/315/commits/680b6411b8bba8a92647deaea967afb22c37fc3f) in the original PR, but forgot the constant.